### ISSUE TITLE
StyleCopTester doesn't take stylecop.json into account.

### ISSUE THREAD

**Issue by vyrp** [DotNetAnalyzers/StyleCopAnalyzers#2977]
I noticed that the StyleCopTester command line program doesn't take stylecop.json into account.

For example, for this snippet, no violations are found in Visual Studio:
```
// <copyright file="Program.cs" company="FooBar">
// Copyright (c) FooBar. All rights reserved.
// </copyright>
```
With stylecop.json being:
```
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "documentationRules": {
      "companyName": "FooBar"
    }
  }
}
```
No errors in VS:
![image](https://user-images.githubusercontent.com/3949802/60762261-2740f480-a010-11e9-80e3-34b1eda26c34.png)

### Actual behaviour

However, when I run StyleCopTester, I get:
```
Loaded solution in 3324ms
Found 4 diagnostics in 2066ms
  CS8019: 2 instances
  SA1636: 2 instances
```
One of the SA1636 violations is (retrieved using StyleCopTester's `/log` flag):
```
some\path\Program.cs(1,4): warning SA1636: The file header copyright text should match the copyright text from the settings.
```
(The other is in AssemblyInfo.cs.)

### Expected behaviour

It would be nice if VS and StyleCopTester produced the same diagnostics:
```
Loaded solution in 3308ms
Found 2 diagnostics in 2063ms
  CS8019: 2 instances
```

**Comment by vyrp** [comment]
I actually found out how to fix this issue, so I will be creating a PR shortly.
